docs/release-notes/index.md

```diff
@@ -21,6 +21,10 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [logstash-next-fixes]
 % *
 
+## 9.1.9 [logstash-9.1.9-release-notes]
+
+No user-facing changes in Logstash core and plugins.
```

Comment:
Probably call out the logging improvement https://github.com/elastic/logstash/commit/33a62d1cc1fd3010bc88dcba870cb2dfb91c3269 ?

Reply:
Fixed, and nice you have implemented the branch validation 👍